src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -26,25 +26,27 @@
      {:color (:button-blue style/colors)})
    :render
    (fn [{:keys [props]}]
-     (let [{:keys [color icon href disabled? onClick text style class-name]} props]
+     (let [{:keys [color icon href disabled? onClick text style type class-name]} props]
        [:a {:className (or class-name "button")
-            :style {:display "inline-block"
-                    :backgroundColor color
-                    :WebkitFilter (when disabled? "grayscale()")
-                    :cursor (when disabled? "default")
-                    :color "white" :fontWeight 500
-                    :borderRadius 2 :padding (if icon "0.7em" "0.7em 1em")
-                    :fontFamily (when icon "fontIcons")
-                    :fontSize (when icon "80%")
-                    :textDecoration "none"}
+            :style (merge
+                     {:display "inline-block"
+                      :backgroundColor color
+                      :WebkitFilter (when disabled? "grayscale()")
+                      :cursor (when disabled? "default")
+                      :color "white" :fontWeight 500
+                      :borderRadius 2 :padding (if icon "0.7em" "0.7em 1em")
+                      :fontFamily (when icon "fontIcons")
+                      :fontSize (when icon "80%")
+                      :textDecoration "none"}
+                     (if (map? style) style {}))
             :href (or href "javascript:;")
             :onClick (if disabled?
                        #(js/alert (if (string? disabled?) disabled? "This action is disabled"))
                        onClick)
             :onKeyDown (when (and onClick (not disabled?))
                          (common/create-key-handler [:space :enter] onClick))}
         (or text (icons/icon-text icon))
-        (when (= style :add)
+        (when (= type :add)
```

Comment:
I see a few other instances of `:style :add`:
* method_repo/methods_configs_acl.cljs
* workspace/summary/acl_editor.cljs
* workspace/summary/attribute_editor.cljs